app/models/solidus_afterpay/order_component_builder.rb

```diff
@@ -98,8 +98,28 @@ def item(line_item)
         price: ::Afterpay::Components::Money.new(
           amount: line_item.price.to_s,
           currency: line_item.currency
-        )
+        ),
+        preorder: check_date(line_item),
+        estimated_shipment_date: estimated_shipment_date(line_item)
       )
     end
+
+    def estimated_shipment_date_product(line_item)
+      line_item.product.property("estimatedShipmentDate")
```

Comment:
It's just me, but I really don't like `estimatedShipmentDate` as a property name 😄

Reply:
Because that's how it is on the afterpay documentation, it would be easier for people to find out to add a property called `estimatedShipmentDate` than anything else